src/protocol/uniswap_v3/state.rs

```diff
@@ -152,11 +152,14 @@ impl UniswapV3State {
                         new_state.liquidity = state.liquidity;
                         new_state.tick = state.tick;
                         new_state.sqrt_price = state.sqrt_price;
-                        return Err(SimulationError::InsufficientData(GetAmountOutResult::new(
-                            state.amount_calculated.abs().into_raw(),
-                            gas_used,
-                            Box::new(new_state),
-                        )));
+                        return Err(SimulationError::RetryDifferentInput(
+                            "Insufficient data".into(),
```

Comment:
With tycho this is not insufficient data anymore but instead you can just call this one "ticks exceeded".